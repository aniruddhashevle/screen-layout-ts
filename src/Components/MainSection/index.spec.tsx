// #region Global Imports
import * as React from "react";
import { renderWithProviders } from "@Test/Helpers/render";
// #endregion Global Imports

// #region Local Imports
import { MainSection } from "@Components";
// #endregion Local Imports

describe("Components", () => {
    describe("MainSection", () => {

        it("should match snapshot", () => {
            const wrapper = renderWithProviders(<MainSection />);

            expect(wrapper).toMatchSnapshot();
        });
    });
});