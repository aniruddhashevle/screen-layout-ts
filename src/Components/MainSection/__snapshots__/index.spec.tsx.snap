// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components MainSection should match snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fce0a7",
        "height": 600,
        "marginBottom": 10,
        "marginTop": 10,
      },
    ]
  }
/>
`;
